client/src/components/FlipCard.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+
+const FlipCardContainer = styled.div`
+  background-color: transparent;
+  margin: 8vw;
+  height: 300px;
+  width: 250px;
+  border-radius: 12px;
+  display: grid;
+  grid-template-rows: auto auto 3fr;
+`;
+
+const FlipCardInner = styled.div`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  transition: transform 0.8s;
+  transform-style: preserve-3d;
+  :hover {
+    transform: rotateY(180deg);
+  }
+`;
+
+const FlipCardFront = styled.div`
+  position: absolute;
+
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+`;
```

Comment:
you could probably change that to sipmly visibility:hidden

Reply:
Thank you for your Feedback :) the backface-visibility property is set because i need this one specificly for the FlipCard action :)